centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -119,6 +122,18 @@ object Operations {
     storageOptions.getService
   }
 
+  lazy val awsConfiguration: AwsConfiguration = AwsConfiguration(CentaurConfig.conf)
```

Comment:
Why lazy?
Is there a chance it won't be used?

Reply:
I suppose if we will not use aws s3 options for checking. For example we coud use only local file checking or gs.